Tests/CMakeOnly/GNUInstallDirs2Other/CMakeLists.txt

```diff
@@ -0,0 +1,75 @@
+cmake_minimum_required(VERSION 3.2.3)
+
+project(GNUInstallDirsTest)
```

Comment:
These tests do not actually compile any code so they should be able to use the `NONE` option to the `project()` command to avoid enabling languages.  This will make them much faster, especially on IDE generators like Visual Studio.